src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx

```diff
@@ -25,10 +25,18 @@ function FlaggedAccount() {
 
 function InvalidRoutingNumber() {
   return (
-    <p>
-      We couldn’t find a bank linked to this routing number. Please check your
-      bank’s 9-digit routing number and enter it again.
-    </p>
+    <div id="errors" role="alert" aria-atomic="true">
+      <div className="usa-alert usa-alert-error">
+        <div className="usa-alert-body">
+          <div className="usa-alert-text">
+            <p>
+              We couldn’t find a bank linked to this routing number. Please
+              check your bank’s 9-digit routing number and enter it again.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
I think you can just add the `role` and `aria-atomic` attributes to the `AlertBox` component at the bottom, https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx#L64.

Triggering this error is tricky I think because of limitations with our local API setups, but this error shows when the lookup for a bank name from a routing number fails.